src/Compression/ICompressionCodec.h

```diff
@@ -127,8 +134,10 @@ class ICompressionCodec : private boost::noncopyable
     /// If it does nothing.
     virtual bool isNone() const { return false; }
 
+    virtual bool isVectorCodec() const { return false; }
```

Comment:
It is not clear for the average reader what a "vector" is in the context of compression codecs. Needs a code comment.